python/dgl/distributed/kvstore.py

```diff
@@ -0,0 +1,1107 @@
+"""Define distributed kvstore"""
+
+import os
+import time
+import random
+import numpy as np
+
+from . import rpc
+from .constants import get_type_str
+
+from .. import backend as F
+from .._ffi.ndarray import empty_shared_mem
+
+KVSTORE_PULL = 901231
+
+class PartitionPolicy(object):
+    """Wrapper for GraphPartitionBook and RangePartitionBook. 
+
+    We can extend this class to support HeteroGraph in the future.
+
+    Parameters
+    ----------
+    policy_str : str
+        partition policy string, e.g., 'edge' or 'node'.
```

Comment:
this isn't ideal. in the future, we need to support different node types and edge types.